### ISSUE TITLE
Better Triggers

### ISSUE THREAD

**Issue by KYGAS** [Lambda11/pet-skill-replacer-and-user#14]
I mean, the module works and it's cool and all... But srsly, "Castanica Midnight Oil" xD
Replace it with something else, like, literally anything else, Rootbeer is free and wine and other shit xD

**Comment by RobinNorling** [comment]
There is literally a configuration file where you can set whichever item you want

**Comment by KYGAS** [comment]
Yeah, ty cpt. Obvious. I am very well aware of that and I've changed it for myself. However, it might not be as simple for many other people.
